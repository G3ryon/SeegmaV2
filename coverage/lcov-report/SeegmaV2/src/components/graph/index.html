
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for SeegmaV2/src/components/graph</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../../../../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../../../../index.html">All files</a> SeegmaV2/src/components/graph</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">2.54% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>3/118</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">0% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>0/38</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">0% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>0/48</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">2.63% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>3/114</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
    </div>
    <div class='status-line low'></div>
    <div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file low" data-value="graph.js"><a href="graph.js.html">graph.js</a></td>
	<td data-value="2.04" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 2%"></div><div class="cover-empty" style="width: 98%"></div></div>
	</td>
	<td data-value="2.04" class="pct low">2.04%</td>
	<td data-value="49" class="abs low">1/49</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="10" class="abs low">0/10</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="18" class="abs low">0/18</td>
	<td data-value="2.13" class="pct low">2.13%</td>
	<td data-value="47" class="abs low">1/47</td>
	</tr>

<tr>
	<td class="file low" data-value="graphDisp.js"><a href="graphDisp.js.html">graphDisp.js</a></td>
	<td data-value="3.7" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 3%"></div><div class="cover-empty" style="width: 97%"></div></div>
	</td>
	<td data-value="3.7" class="pct low">3.7%</td>
	<td data-value="27" class="abs low">1/27</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="4" class="abs low">0/4</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="12" class="abs low">0/12</td>
	<td data-value="4" class="pct low">4%</td>
	<td data-value="25" class="abs low">1/25</td>
	</tr>

<tr>
	<td class="file low" data-value="settings.js"><a href="settings.js.html">settings.js</a></td>
	<td data-value="2.38" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 2%"></div><div class="cover-empty" style="width: 98%"></div></div>
	</td>
	<td data-value="2.38" class="pct low">2.38%</td>
	<td data-value="42" class="abs low">1/42</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="24" class="abs low">0/24</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="18" class="abs low">0/18</td>
	<td data-value="2.38" class="pct low">2.38%</td>
	<td data-value="42" class="abs low">1/42</td>
	</tr>

</tbody>
</table>
</div>
                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank">istanbul</a>
                at Thu Mar 18 2021 09:42:36 GMT+0100 (GMT+01:00)
            </div>
        </div>
        <script src="../../../../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../../../../sorter.js"></script>
        <script src="../../../../block-navigation.js"></script>
    </body>
</html>
    