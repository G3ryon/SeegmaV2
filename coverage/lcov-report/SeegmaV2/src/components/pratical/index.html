
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for SeegmaV2/src/components/pratical</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../../../../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../../../../index.html">All files</a> SeegmaV2/src/components/pratical</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">80.18% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>89/111</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">50% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>35/70</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">83.02% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>44/53</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">83.84% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>83/99</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
    </div>
    <div class='status-line high'></div>
    <div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file empty" data-value="charts.js"><a href="charts.js.html">charts.js</a></td>
	<td data-value="0" class="pic empty">
	<div class="chart"><div class="cover-fill" style="width: 0%"></div><div class="cover-empty" style="width: 100%"></div></div>
	</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	<td data-value="0" class="pct empty">0%</td>
	<td data-value="0" class="abs empty">0/0</td>
	</tr>

<tr>
	<td class="file medium" data-value="datumPicker.js"><a href="datumPicker.js.html">datumPicker.js</a></td>
	<td data-value="50" class="pic medium">
	<div class="chart"><div class="cover-fill" style="width: 50%"></div><div class="cover-empty" style="width: 50%"></div></div>
	</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="6" class="abs medium">3/6</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="2" class="abs high">2/2</td>
	<td data-value="33.33" class="pct low">33.33%</td>
	<td data-value="3" class="abs low">1/3</td>
	<td data-value="60" class="pct medium">60%</td>
	<td data-value="5" class="abs medium">3/5</td>
	</tr>

<tr>
	<td class="file high" data-value="input.js"><a href="input.js.html">input.js</a></td>
	<td data-value="86.36" class="pic high">
	<div class="chart"><div class="cover-fill" style="width: 86%"></div><div class="cover-empty" style="width: 14%"></div></div>
	</td>
	<td data-value="86.36" class="pct high">86.36%</td>
	<td data-value="22" class="abs high">19/22</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="10" class="abs medium">5/10</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="8" class="abs high">8/8</td>
	<td data-value="88.89" class="pct high">88.89%</td>
	<td data-value="18" class="abs high">16/18</td>
	</tr>

<tr>
	<td class="file high" data-value="modal.js"><a href="modal.js.html">modal.js</a></td>
	<td data-value="83.33" class="pic high">
	<div class="chart"><div class="cover-fill" style="width: 83%"></div><div class="cover-empty" style="width: 17%"></div></div>
	</td>
	<td data-value="83.33" class="pct high">83.33%</td>
	<td data-value="6" class="abs high">5/6</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="4" class="abs medium">2/4</td>
	<td data-value="80" class="pct high">80%</td>
	<td data-value="5" class="abs high">4/5</td>
	<td data-value="83.33" class="pct high">83.33%</td>
	<td data-value="6" class="abs high">5/6</td>
	</tr>

<tr>
	<td class="file high" data-value="radioButton.js"><a href="radioButton.js.html">radioButton.js</a></td>
	<td data-value="100" class="pic high">
	<div class="chart"><div class="cover-fill cover-full" style="width: 100%"></div><div class="cover-empty" style="width: 0%"></div></div>
	</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="6" class="abs high">6/6</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="4" class="abs high">4/4</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="3" class="abs high">3/3</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="5" class="abs high">5/5</td>
	</tr>

<tr>
	<td class="file high" data-value="select.js"><a href="select.js.html">select.js</a></td>
	<td data-value="100" class="pic high">
	<div class="chart"><div class="cover-fill cover-full" style="width: 100%"></div><div class="cover-empty" style="width: 0%"></div></div>
	</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="8" class="abs high">8/8</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="5" class="abs high">5/5</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="8" class="abs high">8/8</td>
	</tr>

<tr>
	<td class="file high" data-value="tilesView.js"><a href="tilesView.js.html">tilesView.js</a></td>
	<td data-value="87.88" class="pic high">
	<div class="chart"><div class="cover-fill" style="width: 87%"></div><div class="cover-empty" style="width: 13%"></div></div>
	</td>
	<td data-value="87.88" class="pct high">87.88%</td>
	<td data-value="33" class="abs high">29/33</td>
	<td data-value="55" class="pct medium">55%</td>
	<td data-value="20" class="abs medium">11/20</td>
	<td data-value="85" class="pct high">85%</td>
	<td data-value="20" class="abs high">17/20</td>
	<td data-value="96.43" class="pct high">96.43%</td>
	<td data-value="28" class="abs high">27/28</td>
	</tr>

<tr>
	<td class="file medium" data-value="timeGroup.js"><a href="timeGroup.js.html">timeGroup.js</a></td>
	<td data-value="63.33" class="pic medium">
	<div class="chart"><div class="cover-fill" style="width: 63%"></div><div class="cover-empty" style="width: 37%"></div></div>
	</td>
	<td data-value="63.33" class="pct medium">63.33%</td>
	<td data-value="30" class="abs medium">19/30</td>
	<td data-value="36.67" class="pct low">36.67%</td>
	<td data-value="30" class="abs low">11/30</td>
	<td data-value="66.67" class="pct medium">66.67%</td>
	<td data-value="9" class="abs medium">6/9</td>
	<td data-value="65.52" class="pct medium">65.52%</td>
	<td data-value="29" class="abs medium">19/29</td>
	</tr>

</tbody>
</table>
</div>
                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank">istanbul</a>
                at Thu Mar 18 2021 09:42:36 GMT+0100 (GMT+01:00)
            </div>
        </div>
        <script src="../../../../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../../../../sorter.js"></script>
        <script src="../../../../block-navigation.js"></script>
    </body>
</html>
    