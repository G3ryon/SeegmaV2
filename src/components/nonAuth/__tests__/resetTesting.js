import 'react-native';
import React from 'react';

import { act } from 'react-test-renderer';

import { fireEvent, render, waitFor } from '@testing-library/react-native';
import Reset from '../reset.js';
import { ApplicationProvider, Layout, Button, Text } from '@ui-kitten/components';
import { mapping, light as lightTheme } from '@eva-design/eva';

// Note: test renderer must be required after react-native.
//import renderer from 'react-test-renderer';

jest.useFakeTimers()

test('Reset renders correctly', async () => {
    const result = render(
      <ApplicationProvider mapping={mapping} theme={lightTheme}><Reset /></ApplicationProvider>
    );
    await act(async () => {expect(result).toMatchSnapshot();})
  });