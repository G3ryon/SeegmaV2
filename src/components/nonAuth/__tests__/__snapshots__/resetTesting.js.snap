// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reset renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View>
    <Image
      source={1}
    />
    <View
      style={Object {}}
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#F7F9FC",
              "borderColor": "#FF3D71",
              "borderRadius": 4,
              "borderWidth": 1,
              "minHeight": 40,
              "paddingHorizontal": 8,
              "paddingVertical": 7,
            },
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "width": "100%",
            },
          ]
        }
      >
        <TextInput
          allowFontScaling={true}
          appearance="default"
          editable={true}
          onBlur={[Function]}
          onChange={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onValidation={[Function]}
          placeHolder="Mail"
          placeholderTextColor="#8F9BB3"
          readonly={false}
          rejectResponderTermination={true}
          required={true}
          secureTextEntry={false}
          size="medium"
          status="danger"
          style={
            Array [
              Object {
                "color": "#222B45",
                "fontFamily": "System",
                "fontSize": 15,
                "fontWeight": "normal",
                "marginHorizontal": 8,
              },
              Object {
                "flexBasis": "auto",
                "flexGrow": 1,
                "flexShrink": 1,
              },
              null,
              undefined,
            ]
          }
          type="text"
          underlineColorAndroid="transparent"
          value="juju_beber@gmail.com"
        />
      </View>
      <View
        style={
          Array [
            Object {
              "marginTop": 4,
            },
            Object {
              "alignItems": "center",
              "flexDirection": "row",
            },
          ]
        }
      >
        <Text
          appearance="default"
          category="p1"
          style={
            Array [
              Object {
                "color": "#222B45",
                "fontFamily": "System",
                "fontSize": 15,
                "fontWeight": "400",
              },
              Array [
                Object {
                  "color": "#FF3D71",
                  "fontFamily": "System",
                  "fontSize": 12,
                  "fontWeight": "400",
                },
                Object {
                  "textAlign": "left",
                },
              ],
            ]
          }
        >
          It's not a valid e-mail
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "rgba(143, 155, 179, 0.24)",
          "borderColor": "rgba(143, 155, 179, 0.24)",
          "borderRadius": 4,
          "borderWidth": 1,
          "flexDirection": "row",
          "justifyContent": "center",
          "minHeight": 40,
          "minWidth": 40,
          "opacity": 1,
          "paddingHorizontal": 10,
          "paddingVertical": 12,
        }
      }
    >
      <Text
        appearance="default"
        category="p1"
        style={
          Array [
            Object {
              "color": "#222B45",
              "fontFamily": "System",
              "fontSize": 15,
              "fontWeight": "400",
            },
            Object {
              "color": "rgba(143, 155, 179, 0.48)",
              "fontFamily": "System",
              "fontSize": 14,
              "fontWeight": "bold",
              "marginHorizontal": 10,
            },
          ]
        }
      >
        Submit
      </Text>
    </View>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#3366FF",
          "borderColor": "#3366FF",
          "borderRadius": 4,
          "borderWidth": 1,
          "flexDirection": "row",
          "justifyContent": "center",
          "minHeight": 40,
          "minWidth": 40,
          "opacity": 1,
          "paddingHorizontal": 10,
          "paddingVertical": 12,
        }
      }
    >
      <Text
        appearance="default"
        category="p1"
        style={
          Array [
            Object {
              "color": "#222B45",
              "fontFamily": "System",
              "fontSize": 15,
              "fontWeight": "400",
            },
            Object {
              "color": "#FFFFFF",
              "fontFamily": "System",
              "fontSize": 14,
              "fontWeight": "bold",
              "marginHorizontal": 10,
            },
          ]
        }
      >
        I remembered my password
      </Text>
    </View>
  </View>
</View>
`;
